### PR TITLE
build/tools/amebasmart/gnu_utility: Enable LOGUART 1.5Mbps as wakeup …

### PR DESCRIPTION
**Pull request by zhongnuo-tang**
…interrupt src

1. Enable LOGUART 1.5Mbps as wake up interrupt src
2. Dynamic on/off XTAL clk according to LOGUART baudrate
3. LOGUART 1.5M bps increase power consumption by 1.2mA

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@ewoodev @gSahitya-samsung Could you check this?

File: build/tools/amebasmart/gnu_utility/km0_km4_app_RELEASE_NOTE.txt [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* == "version" + "Realtek git version" + "compile date" + "compile time" == */
 
+== version ecc25123e7 2024/10/15-11:37:48 ==
+1. Enable LOGUART 1.5Mbps as wake up interrupt src
```
Comment (gSahitya-samsung) [inline build/tools/amebasmart/gnu_utility/km0_km4_app_RELEASE_NOTE.txt:4]: Verified board is waking up with baudrate of 1.5Mbps

**Review by gSahitya-samsung — APPROVED** [review]

**Comment by sunghan-chang** [comment]
@zhongnuo-tang Could you check conflicts?

**Comment by zhongnuo-tang** [comment]
> @zhongnuo-tang Could you check conflicts?

Dear @sunghan-chang，

I have rebased it, conflict has been resolved, please check again.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**